Source/Models/Constants.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Constants.swift
+//  Cotter
+//
+//  Created by Albert Purnama on 2/29/20.
+//
+
+import Foundation
+
+public struct CotterConstants {
+    // MARK: - List of verification methods
+    static let MethodPIN = "PIN"
+    static let MethodBiometric = "BIOMETRIC"
```

Comment:
question to ponder: do we need to store this information on the client?

i just don't see any of these being used at all in this PR. also i feel like client doesn't need to know about the verification methods offered by Cotter, argue me otherwise tho if you think it's needed tho.

Reply:
just leave this for now, I think it's needed for clients to check if certain verification method is available or not.